src/style.css

```diff
@@ -0,0 +1,117 @@
+/* stylelint-disable color-hex-case */
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  background: white;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+}
+
+.card__image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: #060B35;
+  margin: 0 0 4px;
+  font-style: medium;
+}
+
+.card__code {
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.card__review {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 24px;
+
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n+1),
+.stars--2 .stars__star:nth-child(-n+2),
+.stars--3 .stars__star:nth-child(-n+3),
+.stars--4 .stars__star:nth-child(-n+4),
+.stars--5 .stars__star:nth-child(-n+5) {
+  background-image: url(./images/star-active.svg);
+}
+
+.card__reviews {
+  font-size: 10px;
+  line-height: 14px;
+  color: #060B35;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.price-label {
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.price {
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: 700;
+}
+
+.card__link {
+  text-decoration: none;
+  display: flex;
+  justify-content: center;
+  box-sizing: border-box;
+  height: 40px;
+  border: 1px solid #00ACDC;
```

Comment:
you used this color many times
create a variable
create variables for repeated values